src/main/resources/stsjorbsmodResources/localization/spa/JorbsMod-Card-Strings.json

```diff
@@ -197,7 +197,7 @@
   },
   "stsjorbsmod:DemonicCoup": {
     "NAME": "Golpe Diabólica",
-    "DESCRIPTION": "stsjorbsmod:Efímera. NL Gana !M! Ritual. NL stsjorbsmod:Expirar."
+    "DESCRIPTION": "Retiene. NL Al Retener esta carta, Gana !stsjorbsmod:MetaMagic! Ritual. NL Gana !M! Ritual stsjorbsmod:Expirar."
```

Comment:
@acourrau Noticing this translation is missing an `NL` before `stsjorbsmod:Expirar`. Also, we've swapped the order of `when retained...` and `gain ritual` texts.

Reply:
Will fix.